core/src/main/resources/hudson/model/AbstractBuild/index_pt_BR.properties

```diff
@@ -33,3 +33,4 @@ Downstream\ Builds=builds filho
 none=nenhum
 Took=Demorou
 Build=Build
+on=ativo
```

Comment:
In this case this translation would be incorrect. Here's where it is used: https://github.com/cpanato/jenkins/blob/2f11dc2fea19c65b0b9a399531ddf775bbe2e55a/core/src/main/resources/hudson/model/AbstractBuild/index.jelly#L45

The "on" is used in a sentence like "Took 10ms on ...."

Probably "em" would be a better translation.

Reply:
done